syntax = "proto3";

package secret;

// ¹«Ë½Ô¿¶Ô
message KeyPair{
  // ¹«Ô¿
  bytes pub_key = 1;

  // Ë½Ô¿
  bytes priv_key = 2;
}

message SecretInfo{
  repeated KeyPair keypair = 1;
}